kgb_rules/status_rules.py

```diff
@@ -23,9 +19,8 @@ def check_status_length(status_line):
     >>> check_status_length("TOO_LONG" * 10)
     'Limit status to 50 characters'
     """
-    if not _is_merge(status_line):
-        if len(status_line) > MAX_STATUS_LENGTH:
-            return "Limit status to {0} characters".format(MAX_STATUS_LENGTH)
+    if len(status_line) > MAX_STATUS_LENGTH:
+        return "Limit status to {0} characters".format(MAX_STATUS_LENGTH)
 
 
 def check_status_formatting(status_line):
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AFunctionComplexity)